foursight_core/app_utils.py

```diff
@@ -59,7 +65,47 @@ class AppUtilsCore:
     TRIM_ERR_OUTPUT = 'Output too large to provide on main page - see check result directly'
     LAMBDA_MAX_BODY_SIZE = 5500000  # 6Mb is the "real" threshold
 
+    # dmichaels/2022-07-20: New to apply the IDENTITY values globally to os.environ (C4-826).
+    def apply_identity_globally(self):
+
+        # This maps key names in the global application configuration (GAC) to names used here.
+        IDENTITY_KEY_MAP = {
+            "ENCODED_AUTH0_CLIENT": "CLIENT_ID",
+            "ENCODED_AUTH0_SECRET": "CLIENT_SECRET",
+            "ENCODED_S3_ENCRYPT_KEY_ID": "S3_ENCRYPT_KEY_ID",
+            "ENCODED_ES_SERVER": "ES_HOST",
```

Comment:
Not a thing for now, but we might consider adding `FOURSIGHT_ES_SERVER` so these values can be different (they are right now as it is set via argument in 4dn-cloud-infra).